src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,11 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
```

Comment:
unnecessary, bc you get type that you always will have user
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
```